### PR TITLE
Login / Logout form

### PR DESCRIPTION
**Pull request by SylvG**
- Login form
- Login form on the page top
- First steps for the admin zone index
- Logout function

depends on #12

### CODE REVIEW

File: src/Wanimo/Mowlkky/BackBundle/Resources/views/Login/login.html.twig [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                   placeholder="{{ 'back.login.login'|trans() }}" class="form-control" required autofocus />
+
+            <label for="password" class="sr-only">{{ 'back.login.password'|trans() }}:</label>
+            <input type="password" id="password" name="_password" class="form-control" placeholder="{{ 'back.login.password'|trans() }}" required />
+
+            {#
+                If you want to control the URL the user
+                is redirected to on success (more details below)
+                <input type="hidden" name="_target_path" value="/account" />
+            #}
```
Comment (SylvG) [inline src/Wanimo/Mowlkky/BackBundle/Resources/views/Login/login.html.twig]: Oups !
I forgot this :)

File: src/Wanimo/Mowlkky/CoreBundle/Security/UserAuthentication.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+     */
+    public function getPassword()
+    {
+        return (string) $this->user->getPassword()->getValue();
+    }
+
+    /**
+     * @return string|null The salt
+     */
+    public function getSalt()
```
Comment (HitoTech) [inline src/Wanimo/Mowlkky/CoreBundle/Security/UserAuthentication.php]: Shouldn't you had add a ``: string`` here thanks to PHP7 to specify what to return ? The same question applies for the rest of the entity.

File: src/Wanimo/Mowlkky/CoreBundle/Security/UserAuthentication.php [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    /**
+     * Removes sensitive data from the user.
+     *
+     * This is important if, at any given point, sensitive information like
+     * the plain-text password is stored on this object.
+     */
+    public function eraseCredentials()
+    {
+
```
Comment (HitoTech) [inline src/Wanimo/Mowlkky/CoreBundle/Security/UserAuthentication.php]: Coming in another PR ?

**Comment by SylvG** [comment]
@HitoTech all your comments are about the Symfony interface we must implement. So we have no choice about it.

**Pull request merged**